Networking/Networking/Remote/MediaRemote.swift

```diff
@@ -18,11 +18,7 @@ public protocol MediaRemoteProtocol {
                                            pageNumber: Int,
                                            pageSize: Int,
                                            completion: @escaping (Result<[WordPressMedia], Error>) -> Void)
-    func uploadMedia(for siteID: Int64,
-                     productID: Int64,
-                     mediaItems: [UploadableMedia],
-                     completion: @escaping (Result<[Media], Error>) -> Void)
-    func uploadMediaToWordPressSite(siteID: Int64,
+    func uploadMedia(siteID: Int64,
                                     productID: Int64,
                                     mediaItem: UploadableMedia,
                                     completion: @escaping (Result<WordPressMedia, Error>) -> Void)
```

Comment:
Super nit: indentation needs to be updated here.

Reply:
Good catch. Done in https://github.com/woocommerce/woocommerce-ios/pull/14537/commits/de07dbdb5e199fa0fe51a922c277dc4bf935956f